CONTRIBUTING.md

```diff
@@ -33,3 +33,24 @@ npm install
 npm test
 ```
 
+# Coding Guidelines
+
+## License header
+
+All source files must begin with a comment of the below header:
+
+```go
+// Copyright 2017 Google Inc.
```

Comment:
- Replaced 2017 with {YEAR}
- .html and .css files do need it - I'll mention that